tests/binary/test_inline_single_instance.py

```diff
@@ -0,0 +1,11 @@
+import delegator
```

Comment:
why a python test?

Reply:
No particular reason other than it being easier to write than bash.